### ISSUE TITLE
Poses issue when adding fixed_pose engine in the gen_grid_map.ipynb

### ISSUE THREAD

**Issue by GimpelZhang** [boschresearch/pcg_gazebo#121]
Hi all,

hope this issue will not bother you, since this project is still under development. 

But in the gen_grid_map.ipynb tutorial, when adding fixed_pose engine in this cell of code:

```
world_gen.add_engine(
    engine_name='fixed_pose',
    tag='gp_engine',
    models=['box_floor'],
    poses=[[0, 0, -0.005, 0, 0, 0]])
```

it seems that the "-0.005" z-direction position is not enough to avoid collision for adding the "box_placement" random_pose engine. Therefore no model except for the box_floor occurs in the world, and the following message shows up when requesting the grid map:

`2020-04-14 22:13:05,230 | WARNING | occupancy | List of models is empty, cannot compute occupancy grid`

**Comment by musamarcusso** [comment]
Hi @GimpelZhang , thanks for reporting that, I just pushed the fix to #122. Once the pipeline tests it, I will merge it into `master`.

**Comment by GimpelZhang** [comment]
Great! Thanks.
